zipline/testing/fixtures.py

```diff
@@ -527,7 +527,7 @@ def init_class_fixtures(cls):
         cls.sim_params = cls.make_simparams()
 
 
-class WithTradingSessions(WithTradingCalendars):
+class WithTradingSessions(WithTradingCalendars, WithDefaultDateBounds):
```

Comment:
The order here shouldn't matter; both `WithTradingCalendars` and `WithDefaultDateBounds` are direct subclasses of `object`.  The tests are currently failing because another class is manually inheriting from both `WithTradingSessions` and `WithDefaultDateBounds`.  The right fix is to remove the duplication at that use-site.